x/programs/runtime/dependencies.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 type Runtime interface {
+	// Create initializes and calls the init function.
+	Create(context.Context, []byte) (uint64, error)
```

Comment:
I don't believe this is actually a necessary part of the runtime interface. Once the runtime is initialized then the user can make whatever call they want. My understanding is that a programID would be the result of a transaction generated by the VM.